src/js/components/DataChart/utils.js

```diff
@@ -78,7 +78,13 @@ export const createDateFormat = (firstValue, lastValue, full) => {
   ) {
     const delta = Math.abs(endDate - startDate);
     let options;
-    if (delta < 60000)
+    if (delta === 0)
+      // equal 0 - single data point
+      options = {
+        month: full ? 'short' : 'numeric',
+        day: 'numeric',
+      };
+    else if (delta < 60000)
```

Comment:
```suggestion
    if (delta < 60000)
```
Lets revert this change back, I don't think we need to adjust this